src/main/java/com/uid2/admin/Main.java

```diff
@@ -161,10 +164,9 @@ public void run() {
 
             //UID2-575 setup a job dispatcher that will write private site data periodically if there is any changes
             //check job for every minute
-            JobDispatcher.getInstance().start(1000 * 60, 3);
             PrivateSiteDataSyncJob job = new PrivateSiteDataSyncJob(config, writeLock);
-            JobDispatcher.getInstance().enqueue(job);
-            JobDispatcher.getInstance().executeNextJob(3);
+            jobDispatcher.enqueue(job);
+            jobDispatcher.executeNextJob();
```

Comment:
Why is this changed? Not sure why it did 3 before either though.

Reply:
changing it from singleton to non-singleton

3 was there because you had to pass in `maxRetries`

Reply:
ah right